src/uucore/lib.rs

```diff
@@ -1,13 +1,9 @@
 extern crate wild;
 
-pub fn args() -> Box<Iterator<Item=String>> {
+pub fn args() -> impl Iterator<Item=String> {
```

Comment:
Is this supported by Rust 1.22?

Reply:
No, `impl Trait` support didn't land until 1.26 (see <https://blog.rust-lang.org/2018/05/10/Rust-1.26.html>). I'm happy to revert it to using boxing. It does compile, test, and run correctly with the boxing version, though it's likely slightly (probably undetectably) less performant.

But, FYI, in testing this, I can't get the repo to compile using the "1.22.1" toolchain anyway. There are lots of "f32::from_bits(...)" and similar errors.

To make the backwards compatibility more explicit, it might be a good idea to modify the AppVeyor CI to test back to the minimum version that you'd like to support. There's an example configuration being used by [globwalk](https://github.com/rivy/rust.globwalk/blob/master/appveyor.yml) that I could modify into a PR which tests more widely if that's a help.

Reply:
I've fine with bumping up to 1.26.  I want to use `impl Trait` too (lol).  If you could modify that in `README.md` I'll merge this as is.

Reply:
Sure, I'll be happy to update the `README.md`. While I'm doing that, I'll also double-check that it actually compiles with 1.26 as well.

Reply:
Unfortunately, rust@1.26 isn't enough. It looks like rust@1.27 is the actual minimum needed.

Several external crates, which were upgraded to newer versions in prior commits, are using `from_bits` which causes compile-time errors. I can't find a definitive source (eg, CHANGELOG) that mentions when `from_bits` was landed into stable mainline, but I've found a discussion that says rust@1.27 is needed (see <https://github.com/starkat99/half-rs/issues/11#issuecomment-404700575>). And I found that to be the case, as well.

Compilation and testing work fine with rust@1.27 (and I added a specific test to the Travis CI configuration for the minimum rust version). There is a Travis CI caveat (see below). Would you like me to change the minimum to 1.27? The alternative is to revert all the external crate upgrades which are requiring 1.27.

Notably, Travis CI builds are failing because of use of "Sphinx" 1.8.0 during install testing. Older builds succeeded using Sphinx@1.7.8. I have no experience with that build process, and I don't have a suggested solution (other than to cut it out).

There's also a bit of old code cruft that can be taken out to avoid all of the build warnings.

Reply:
@Arcterus , I've got a branch with rust@1.27 and revised / working Travis CI testing at <https://github.com/rivy/rust.coreutils/tree/fix.ci>, specifically <https://github.com/rivy/rust.coreutils/commit/deead6f672925cab72e319770f9214b1cd07741a>.

AppVeyor CI was failing for odd reasons, so I remodeled it based upon a newer template (see <https://github.com/starkat99/appveyor-rust/blob/master/appveyor.yml>). 

The Travis CI build is at <https://travis-ci.org/rivy/rust.coreutils/builds/432010513>.

The AppVeyor CI build is at <https://ci.appveyor.com/project/rivy/rust-coreutils/build/2%20~%20fix.ci>.

I can add all or part of it to this PR... let me know how you'd like me to proceed.